doc/previews.md

```diff
@@ -0,0 +1,17 @@
+# Quick previews
+
+## VSCode plugin
+
+![VSCode Plugin Preview](./img/vscode-2023-03-10.mp4)
```

Comment:
I haven't tested this before, but perhaps it will work:

```suggestion
<video src='./img/vscode-2023-03-10.mp4.mp4' width=180/>
```

Reply:
If it doesn't work, we can try uploading to `githubusercontent` itself. Here's an example of how that would look: https://github.com/mfitzp/diffcast/blob/master/README.md

Reply:
yeah, the video tag did not work

Reply:
Perhaps we should try to compress the videos. The one in the link has 38 seconds and only 2.14 MB, and the quality is ok